docs/src/index.md

```diff
@@ -1,6 +1,45 @@
 # Javis
 
-Javis.jl (Julia mAthematical VISualization) is a visualization and animation package that builds on top of [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://Wikunia.github.io/Javis.jl/stable)
```

Comment:
This file is intended to provide a different introduction than a Readme does. For example more examples and a tutorial for first steps.

Reply:
That makes sense. I will reset it to what you have had before. Do you think I can add anything to this page?

Reply:
It should probably include some examples so you could try to add an example with LaTeX bit currently all actual functionality is inside branches so later is probably more interesting.

Reply:
As #6 got merged there are some new features we want to have in a tutorial. Interested @sudomaze

Reply:
Yeah, sure. I will finish #21 then I will be down to work on adding the examples to the documentation.